### PR TITLE
We are Xdev! Blog Post

### PR DESCRIPTION
**Pull request by kmpaul**
This is a draft of the "We are Xdev!" blog post for the Xdev website.  I am looking for improvements, though, so feel free to fork my fork and suggest changes that way.  Or just review the code and tell me improvements that I can make.  I kinda lost some steam at the end and need a new pair of eyes to tell me where I lost the theme...

### CODE REVIEW

**Comment by jukent** [comment]
Reads great! Thanks for putting this together

**Comment by kmpaul** [comment]
Okay.  I've given the draft a few more read throughs.  If I can get 2 approving reviews, I'll post this blog today.

@NCAR/xdev

File: site/posts/we-are-xdev.md [hunk 1]
```diff
@@ -1,39 +1,75 @@
...
-_exemplify_ modern best practices in the development and use of these resources.  We took
-guidance from remarkably effective communities like [Pangeo](https://pangeo.io), which continues
-to show the enormous potential of Open Source development when coupled with modern software best
-practices, and we started to develop a plan of how to shift, albeit slowly, both NCAR and the
+More than a year ago, [Matthew Long](http://www.cgd.ucar.edu/staff/mclong/) and
+[I](https://staff.ucar.edu/users/kpaul) started shopping around the idea that a
+lot of what we do here at [NCAR](https://ncar.ucar.edu/), as well as *how* we do
+it, needed a bit of a facelift.  Our pitch was simple: As an institution that
+supports the geoscientific community by providing valuable datasets, supercomputing
+facilities, and computer models, NCAR need's to _exemplify_ modern best practices
```
Comment (andersy005) [inline site/posts/we-are-xdev.md]: NCAR need's to --> NCAR needs to ??

Comment (kmpaul) [reply site/posts/we-are-xdev.md]: Yes.  Good catch.  Thank you!

**Pull request merged**